metrics/mixin/rules/rules.libsonnet

```diff
@@ -91,7 +90,7 @@
           {
             record: 'odf_system_latency_seconds',
             expr: |||
-              avg by (namespace, managedBy, job, service)
+              sum by (namespace, managedBy, job, service)
```

Comment:
Can you explain why changing "avg" to "sum" and removing division by 2 works here?
Seems completely different from original query.